### DISCUSSION TITLE
Converting from Caffe to Mxnet  using  tools/convert_symbol  (scale layer)

### DISCUSSION THREAD

**Discussion by Mypathissional** [apache/incubator-mxnet#15093] [Q&A]
Hi,

I've noticed that one cannot convert the prototxt to a symbol, if there are scale layers if the previous layer was not batch norm layer. So I was wondering which corrections should be done in order to be able to convert the following part 
```
layer {
  name: "res3_1_1_srelu_3"
  type: "Scale"
  bottom: "res3_1_1"
  top: "res3_1_1"
  param {
    lr_mult: 0.0
    decay_mult: 0.0
  }
  param {
    lr_mult: 0.0
    decay_mult: 0.0
  }
  scale_param {
    filler {
      value: 1.0
    }
    bias_term: true
    bias_filler {
      value: -1.0
    }
  }
}
```
Would be grateful for any help :)))

Thanks in advance

**Comment by frankfliu** [comment]
@mxnet-label-bot add [question, operator]
